docs/resources/acmpca_certificate.md

```diff
@@ -274,6 +274,26 @@ Optional:
 
 Import is supported using the following syntax:
 
+In Terraform v1.12.0 and later, the [`import` block](https://developer.hashicorp.com/terraform/language/import) can be used with the `identity` attribute, for example:
+
+```terraform
+import {
+  to = awscc_acmpca_certificate.example
+  identity = {
+    arn                       = "arn"
+    certificate_authority_arn = "certificate_authority_arn"
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+### Identity Schema
+
```

Comment:
The optional `account_id` and `region` identity attributes should also be documented